contracts/root/withdrawManager/WithdrawManagerStorage.sol

```diff
@@ -1,10 +1,51 @@
-pragma solidity ^0.5.5;
+pragma solidity ^0.5.2;
 
 import { Registry } from '../Registry.sol';
 // import { RootChain } from '../RootChain.sol';
-import { ProxyData } from '../../common/misc/ProxyData.sol';
+import { ProxyStorage } from '../../common/misc/ProxyStorage.sol';
 
-contract WithdrawManagerStorage is ProxyData {
+
+contract ExitManagerStorage {
```

Comment:
Why do we need another storage? Let's have these in the withdrawmanagerStorage

Reply:
I am thinking about keeping both separate ExitManager and withdrawManager, let's have some discussion first on the design of it.